src/routes/router.ts

```diff
@@ -54,6 +56,10 @@ RootRouter.use(MaintenanceRouter);
 RootRouter.use("/github", GithubRouter);
 RootRouter.use("/jira", JiraRouter);
 
+RootRouter.post("/webhook/event/:uuid",
```

Comment:
I'd put this under the `GithubRouter` since it is only github webhooks we're handing and not a global webhook handler.

The final url should be something like `github/webhooks/some-id`, but you also need to specify that the uuid is optional since we want to handle the github cloud webhooks as well, so the route should be something like `/webhooks/:uuid?` to make it optional.  If uuid isn't set, it's github.com, if it is set, it's ghe.

Reply:
Moved under `GithubRouter'.  The webhook url for GHE looks like `github/webhooks/some-id` where as for github cloud it is `github/webhooks/cloud`